optimum/intel/openvino/utils.py

```diff
@@ -0,0 +1,25 @@
+#  Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+OV_WEIGHTS_NAME = "ov_model.xml"
```

Comment:
I suggest OV_XML_NAME. See a comment above: the xml file contains the model topology, not the weights.

Reply:
I updated it to `OV_XML_FILE_NAME`, let me know if you have an other prefered suggestion